### PR TITLE
Update code to be compliant with `oteapi-core` v0.0.5

### LINKED ISSUE #36: Update according to new `oteapi-core` plugins API

### PR DESCRIPTION
**Pull request by CasperWA**
Use `create_strategy()` throughout and `load_strategies()`.

For the testing, a new utility function to load the test strategies
under `tests/static/strategies/` has to be added.

Update `oteapi-core` dependency to v0.0.5.

Fixes #36

### CODE REVIEW

File: app/main.py [hunk 1]
```diff
@@ -54,7 +54,7 @@ def create_app() -> FastAPI:
 
 This service is based on:
 
-- **oteapi-core**: v0.0.3
+- **oteapi-core**: v0.0.5
```
Comment (jesper-friis) [inline app/main.py:57]: This looks like a comment that is easy to forget to update. Should we remove it? The needed version of oteapi-core can be found in requirements.txt

**Review by jesper-friis — COMMENTED** [review]
I am ready to approve - but will you still see the comment if I do that?

File: app/main.py (hunk 1 above)
Comment (CasperWA) [reply app/main.py:57]: Sure. Otherwise we can automate it, e.g., using `invoke` - but perhaps that's overkill?

**Review by jesper-friis — APPROVED** [review]

**Comment by CasperWA** [comment]
@jesper-friis updated the section you commented about to instead include a link to `oteapi-core` source code on GitHub.

File: app/main.py (hunk 1 above)
Comment (CasperWA) [reply app/main.py]: See https://github.com/EMMC-ASBL/oteapi-services/pull/39#issuecomment-1032314133

**Pull request merged**